atlp-rwanda/Team-Sostene-E-commerce-bn
# #184651087 Implement Notifications for user to get updated on the status of the products

#### What does this PR do?
Sends notifications to relevant users according to the status of their products
#### Description of Task to be completed?
Provided a product
GIVEN. the product lifecycle changes (added, removed, expired, bought, etc) then  the notification should be sent to the the right user
#### How should this be manually tested?
-Clone the repo and run npm run migrate:all

then login and put token in bearer
you can check your notifications via endpoint /notifications/all with GET method

#### Any background context you want to provide?
Notifications are being sent inApp and via email
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/story/show/184651087
#### Screenshots (if appropriate)
n/a
#### Questions:
